templates/distribution/manifests/networking/kustomization.yaml.tpl

```
@@ -17,6 +16,9 @@ resources:
     {{- if eq .spec.distribution.modules.networking.type "calico" }}
   - {{ print $vendorPrefix "/modules/networking/katalog/tigera/on-prem" }}
   - resources/calico-ns.yml
+      {{- if not (.spec | digAny "kubernetes" "advanced" "kubeProxy" "enabled" true) }}
```

maybe add here the same comment as [cilium-config.env.tpl#R13](https://github.com/sighupio/distribution/pull/459/files#diff-1452617a5c664227166667560563a1f2a1f3f368c217300204953125f2ca44b4R13), just to clarify that we need this condition as it's written
```suggestion
      {{- /* We assume that kubeProxy is enabled by default */}}
      {{- if not (.spec | digAny "kubernetes" "advanced" "kubeProxy" "enabled" true) }}
```